packages/pwa-kit-dev/src/utils/logger.js

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
Lets move this class to the runtime. The dev tool lib is typically just for things being used during development, e.g. config for webpack, dev server, etc. Not for code that will be transpiled down into a pwa's bundle.

Reply:
Would it be better to have a new package for this, so we can pull it into commerce-sdk-react as well?

(Bringing this up here since the PR I'm working on might also want something in a package that could be pulled into both commerce-sdk-react and pwa-kit-runtime 🙂) . But creating a new package is probably something for a separate ticket.